scripts/build_local_binary.sh

```diff
@@ -0,0 +1,30 @@
+# When you run a `make bin` in the terraform-provider-rancher2 repo, it
+# downloads a binary into your local cache from the specified source. To
+# tell terraform to use a local binary, you need to overwrite the one in
+# the cache.
+
+# Add the following to your terraform config file. This tells terraform 
+# to use the local cache https://terraform.io/cli/config/config-file.
+
+```
+terraform {
+  required_providers {
+    rancher2 = {
+      source = "terraform.example.com/local/rancher2"
+      version = "1.0.0"
+    }
+  }
+}
+```
+
+# Compile the local binary
+make bin
+
+# Overwrite the cached binary with your local one
+cp terraform-provider-rancher2 ~/.terraform.d/plugins/terraform.example.com/local/rancher2/1.0.0/darwin_amd64/terraform-provider-rancher2
+
+# Initialize terraform modules
+# terraform init
```

Comment:
Is it intentional to leave these steps commented out? If it's something that may only be needed sometimes, supporting option flags is probably a good idea (not sure if `terraform init / apply` is something we want/need to do as part of script named `build_local_binary.sh`).

Reply:
I include those commands in my own process. I find automating the process from creating the binary to running an apply is super helpful for the efficiency of local testing - with those commands included, I call this script `./run.sh` But for `build_local_binary.sh` we don't need them no. For option flags, I could add optional `-I` and `-a` flags for running an additional init and apply.